blogposts/2022/release-3.38.md

```diff
@@ -0,0 +1,74 @@
+---
+title: 'Sourcegraph 3.38 release'
+publishDate: 2022-03-21T10:00-07:00
+description: 'Sourcegraph 3.38 introduces improved Notebooks, faster Code Intelligence for large repositories, dependencies search, and custom file syntax highlighting.'
+tags: [blog, release]
+slug: 'release/3.38'
+published: true
+heroImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+socialImage: https://storage.googleapis.com/sourcegraph-assets/blog/3.38/sourcegraph-3-38-release.png
+changelogItems:
+  - description: Code Insights will now periodically clean up data series that are not in use. There is a 1 hour grace period where the series can be reattached to a view, after which all of the time series data and metadata will be deleted. This means customers deploying their own instance won't be storing data that has no use.
+    url: https://github.com/sourcegraph/sourcegraph/pull/32094
+    category: Code Insights
+  - description: 'Notebooks from private enterprise instances can now be embedded in external sites by enabling the `enable-embed-route` feature flag. This allows you to embed Notebooks within existing technical documentation to see your documentation and code side-by-side.'
+    url: https://docs.sourcegraph.com/notebooks/notebook-embedding
+    category: Notebooks
+  - description: LSIF upload pages now include a section listing the reasons and related retention policies resulting in an upload being retained and not expired.
+    url: https://github.com/sourcegraph/sourcegraph/pull/30864
+    category: Code Intelligence
+  - description: 'Code monitoring now has a Logs tab enabled as a [Beta feature](https://docs.sourcegraph.com/admin/beta_and_experimental_features). This lets you see recent runs of your code monitors and determine if any notifications were sent or if there were any errors during the run.'
+    url: https://github.com/sourcegraph/sourcegraph/pull/32292
+    category: Admin
```

Comment:
This isn't rendering correctly, the description is blank when I preview the page
<img width="896" alt="Screenshot 2022-03-21 at 16 13 55" src="https://user-images.githubusercontent.com/82806852/159303793-421ac326-dc46-4f7c-a8eb-ba626beac8f9.png">

Reply:
```suggestion
  - description: 'Code monitoring now has a Logs tab enabled as a Beta feature. This lets you see recent runs of your code monitors and determine if any notifications were sent or if there were any errors during the run.'
    url: https://github.com/sourcegraph/sourcegraph/pull/32292
    category: Admin
```

Reply:
Perhaps it doesn't like the second link — trying to fix.